libs/journeys/ui/src/components/Image/Image.spec.tsx

```diff
@@ -23,4 +23,13 @@ describe('Image', () => {
       'random image from unsplash'
     )
   })
+
+  it('should render the default image', () => {
+    const { getByRole } = render(<Image {...block} src={null} alt={'defaultImageIcon'} />)
```

Comment:
Please use the notation below.
```suggestion
    const { getByRole } = render(<Image {...block} src={null} alt="defaultImageIcon" />)
```